ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -135,3 +135,23 @@ Verify Pipeline Run Deployment Is Successful    # robocop: disable
     ${containerStatuses}=  Create List        terminated    terminated
     ...    terminated    terminated    terminated
     Verify Deployment    ${valid_model}  1  1  ${containerNames}    ${podStatuses}    ${containerStatuses}
+
+Delete Pipeline Ran
+    [Documentation]    Delete a pipeline that ran based on name and pipeline. From the left menu select
+    ...                "Data Science Pipelines" -> Runs. In the new page, select the tab "Triggered".
+    ...                The "Delete Pipeline Ran" will search for a line in the grid that match the pipeline name and
+    ...                the run name. Based on that, hit the ... Menu in the row and hit Delete drop down menu.
+    [Arguments]    ${run_name}
+    Click Element    //button[@id="pipelines"]
+    Click Element    //a[text()='Runs']
```

Comment:
You could probably use one of the keywords in this resource file `ods_ci/tests/Resources/Page/Components/Menu.robot`